app/controllers/concerns/paginable.rb

```diff
@@ -18,7 +18,7 @@ def paginable_renderise(partial: nil, controller: params[:controller], action: p
       @paginable_controller = controller
       @paginable_action = action
       @paginable_path_params = path_params
-      merge_query_params(query_params)
+      @paginable_path_params = @paginable_path_params.merge(merge_query_params(query_params))
```

Comment:
merge_query_params(query_params) is already merging query params into params object and since you are merging that into paginable_path_params instance variable, the variable holds effective a copy of params object and therefore becomes useless.
Please, get rid of @paginable_path_params instance variable across the concern and replace with params instead:
```
params = merge_query_params(query_params)).merge(path_params)
```

Reply:
Hmm. they didn't seem to be available. I may have been looking at the wrong thing though.